python/dask_cudf/dask_cudf/io/tests/test_parquet.py

```diff
@@ -443,7 +443,7 @@ def test_create_metadata_file(tmpdir, partition_on):
     dd.assert_eq(ddf1, ddf2)
 
 
-@xfail_dask_expr("dtypes are inconsistent")
+@xfail_dask_expr("Newer dask version needed", lt_version="2024.5.0")
```

Comment:
Would it make sense to update `xfail_dask_expr` to make this the `_default_reason` if we expect to use it commonly?

Reply:
I suppose it could be if `lt_version` is also passed in? Otherwise the message could be wrong/misleading.

Reply:
Oh yeah totally forgot that `lt_version` is optional here 🤦🏼‍♂️ think that suggestion makes sense but also okay with punting off any more refactoring to `xfail_dask_expr` to follow up work since I assume we intend to remove a lot of these markers soon anyways.